app/src/test/java/org/simple/clinic/home/HomeScreenLogicTest.kt

```diff
@@ -56,23 +53,19 @@ class HomeScreenLogicTest {
         uuid = UUID.fromString("5b2136b8-11d5-4e20-8703-087281679aee"),
         name = "CHC Nathana"
     )
-    val loggedInUser = TestData.loggedInUser(
-        uuid = UUID.fromString("751cfb09-92a2-40df-a6b2-b3f82ecd81a1")
-    )
     val date = LocalDate.parse("2018-01-01")
 
-    whenever(userSession.requireLoggedInUser()).thenReturn(Observable.just(loggedInUser))
-    whenever(facilityRepository.currentFacility(loggedInUser)).thenReturn(Observable.just(facility1, facility2))
     whenever(appointmentRepository.overdueAppointmentsCount(date, facility1)) doReturn Observable.just(3)
     whenever(appointmentRepository.overdueAppointmentsCount(date, facility2)) doReturn Observable.just(0)
 
     // when
-    setupController()
+    setupController(facility1)
+    setupController(facility2)
```

Comment:
This does not match what the behaviour in production is in reality. What we should do instead is pass a stream of facilities to `setupController`.

Example:
```kotlin
setupController(Observable.just(facility1, facility2))
```

Reply:
Ah okay,  fixed it 👍🏾